core/storage/base_model/gae_models_test.py

```diff
@@ -137,6 +143,43 @@ class TestVersionedModel(base_models.VersionedModel):
     SNAPSHOT_CONTENT_CLASS = TestSnapshotContentModel
 
 
+class BaseCommitLogEntryModelTests(test_utils.GenericTestBase):
+
+    def test_get_instance_id_raises_not_implemented_error(self):
+        # Raise NotImplementedError as _get_instance_id is to be overwritten
```

Comment:
Prefer encoding this kind of information into the test name instead of through comments, something like:

`def test_base_class_get_instance_id_raises_not_implemented_error`

Reply:
Done

Reply:
Thanks! Since this comment is now redundant, please remove.

Reply:
Please delete this comment.